### ISSUE TITLE
Shouldn't the halos be circular, not elliptical?

### ISSUE THREAD

**Issue by unfa** [SIsilicon/Godot-Lens-Flare-Plugin#6]
Hey! This is amazing, and I'd love to understand better how it works!

That being said - shouldn't the halos be following a circle rather than an ellipse?
Unless the camera is using an anamorphic lens, the optic elements are all circular, so the effect should not be squeezed to display aspect ratio, it should be cropped on the top and bottom.

Could there be a control added to change that?

I guess anamorphic lens flares are a different beast alltogether :)

**Comment by SIsilicon** [comment]
You are correct that the halo should be circular. In fact, the shader has a uniform called `stretch_to_aspect` to control this. It's been a while since I used it, so try playing with the value to see what it does.
The uniform is right here. https://github.com/SIsilicon/Godot-Lens-Flare-Plugin/blob/d718b7064855d176a8051ac7f7371fe471931f62/addons/silicon.vfx.lens_flare/lens_flare_shader.tres#L8
